src/stores/firebase.js

```diff
@@ -73,20 +73,24 @@ export const useFirebaseStore = defineStore("firebase", {
         this.firebaseApp = initializeApp(firebaseConfigProduction);
       }
 
+      this.firestore = initializeFirestore(this.firebaseApp,{});
+      this.auth = initializeAuth(this.firebaseApp,{});
+      this.functions = getFunctions(this.firebaseApp, "europe-west1");
+
+      if (process.env.DEV) {
+        connectFirestoreEmulator(this.firestore, 'localhost', 8080);
+        connectAuthEmulator(this.auth, "http://localhost:9099");
+        connectFunctionsEmulator(this.functions, "localhost", 5001)
+      }
+
       // 🔥 INITIALISE REALTIME DATABASE
       // this.database = getDatabase(firebaseApp);
```

Comment:
are these comments ours? Do we need them? I would prefer deleting all code that we dont need / use, as usually I see that it just keeps piling up.